src/dataviews/histogram-dataview/histogram-data-model.js

```diff
@@ -130,5 +145,37 @@ module.exports = Model.extend({
     return this.get('localTimezone')
       ? this.get('localOffset')
       : this.get('offset');
+  },
+
+  _getStartEnd: function (columnType, aggregation) {
+    var result = null;
+    if (columnType === 'number' && this._startEndCache[columnType] !== null) {
```

Comment:
I feel `this._startEndCache[columnType]` can be moved to a variable, and reused below